tests/test_crossdock.py

```diff
@@ -66,92 +50,6 @@ def tracer():
             PERMUTATIONS.append((s2, s3, sampled))
 
 
-# noinspection PyShadowingNames
```

Comment:
I don't think these changes are correct. The issue previously was that we wanted the tests to run "in principle" on py3, but they couldn't really do anything useful because tchannel required py2.  But now tchannel is supposed to support py3, so the tests below should be fully executed.

Reply:
Great, going to just remove skip part.